default.nix

```diff
@@ -103,6 +115,7 @@ rec {
     };
 
     hsSrc = stdenv.mkDerivation {
+      inherit (locales) LANG LC_ALL LOCALE_ARCHIVE;
```

Comment:
Is there a way to avoid repeating this 3 times and inherit once at the level of `mkSpecDerivation`?

Reply:
Nah, leave it since it is not trivial.